geometry/geodesic-sector-and-ellipse/src/main/java/com/esri/samples/geodesic_sector_and_ellipse/GeodesicSectorAndEllipseController.java

```diff
@@ -63,11 +64,16 @@
   private MarkerSymbol sectorMarkerSymbol;
 
   public void initialize() {
-    // initialize a map to a viewpoint and set it to the map view
-    ArcGISMap map = new ArcGISMap(Basemap.createImagery());
-    center = new Point(-13574921.207495, 4378809.903179, SpatialReference.create
-        (3857));
+    // authentication with an API key or named user is required to access basemaps and other location services
+    String yourAPIKey = System.getProperty("apiKey");
+    ArcGISRuntimeEnvironment.setApiKey(yourAPIKey);
+
+    // create a map with a basemap style and set the initial viewpoint
+    ArcGISMap map = new ArcGISMap(BasemapStyle.ARCGIS_IMAGERY);
+    center = new Point(-13574921.207495, 4378809.903179, SpatialReference.create(3857));
```

Comment:
I think wkid 3857 is WebMercator:
https://developers.arcgis.com/web-map-specification/objects/spatialReference/
So we could use the enum `SpatialReferences.getWebMercator()`:
https://developers.arcgis.com/java/latest/api-reference/reference/com/esri/arcgisruntime/geometry/SpatialReferences.html#getWebMercator()

However, I'm pretty sure web mercator is the default SR, so might be worth leaving it out of the 'new Point` and seeing if it works?

Reply:
Didn't work removing it. Have replaced it for `getWebMercator()`.  I also made a couple of other additional changes to this one - setting the `ViewPoint` on the `MapView` (which still seems to work correctly) and corrected the basemap style to `IMAGERY_STANDARD`. 

Probably one worth an extra check @Rachael-E

Reply:
looks good!